contracts/peg/ICurveRouter.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import {IVolt} from "../volt/IVolt.sol";
+
+interface ICurveRouter {
+    // ---------- View-Only API ----------
+
+    /// @notice reference to the Volt contract used.
+    function volt() external returns (IVolt);
+
+    /// @notice calculate the amount of VOLT out for a given `amountIn` of underlying
+    function getMintAmountOut(uint256 amountIn)
```

Comment:
is the assumption here that there is only one type of input currency per curve router? if not, we'd need to add an address of the token here